docs/docs/edsl.md

```diff
@@ -2,9 +2,9 @@
 nav_order: 5
 ---
 
-# C++ Tile eDSL
+# C++\* Tile eDSL
```

Comment:
Seriously? Let's tone this way down to just entities that actually have lawyers.